GETTING_HIRED.md

```diff
@@ -0,0 +1,36 @@
+# Getting hired
```

Comment:
Should this be "Oh, you where hired!", "Getting hired" implies steps on how to get hired.